ui/app/mainui/AppMain.qml

```diff
@@ -1573,6 +1573,7 @@ Item {
         clip: false
 
         delegate: StatusToastMessage {
+            objectName: "ephemeralNotificationList_Delegate"
```

Comment:
Haven't seen this naming convention before. If it's not a commonly agreed convention, I'd go for a standard one (https://en.wikipedia.org/wiki/Naming_convention_(programming))

Reply:
i changed that @alexjba thanks for pointing on that